shoop/xtheme/engine.py

```diff
@@ -39,7 +40,9 @@ def render(self, *args, **kwargs):
         return self.environment.handle_exception(exc_info, True)
 
     def _postprocess(self, context, content):
-        # TODO: Add a hook here for addons to inject resources without plugins
+        for inject_func in get_provide_objects("xtheme_resource_injection"):
+            if callable(inject_func):
+                inject_func(context)
```

Comment:
Come to think of it though, I think it might be a good idea to also pass `content` to this method. I could see injectors wanting to read the rendered content for some reason or another.